### PR TITLE
Pull Request (Complete Fibo Function)

### LINKED ISSUE #2: Create Fibo.py (Recursion, DP)
- [x] Make Fibo.py.
- [x] Add Recursion Function
- [x] Add DP Function
- [x] Print Function

### PR DESCRIPTION
**Pull request by ProgWon**
Resolved: #2

### CODE REVIEW

File: fibonacci.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    if n == 1 or n == 2:
+        return 1
+
+    if n in dp:
+        return dp[n]
+
+    dp[n] = fibo_dp(n-1, dp) + fibo_dp(n-2, dp)
+
+    return dp[n]
+
```
Comment (Jack2ee-dev) [inline fibonacci.py:29]: There is no getting input for fibo_dp! You should implement the input function

Comment (ProgWon) [reply fibonacci.py:29]: sorry, I'll check it as soon as possible.

Comment (Jack2ee-dev) [reply fibonacci.py:29]: great job!

**Comment by ProgWon** [comment]
I checked it.

**Pull request merged**